src/zcl_timem_object_tr.clas.abap

```diff
@@ -0,0 +1,98 @@
+"! Representation of a program/include object. This object consists of a single part which
+"! this class  will be able to create and return.
+CLASS zcl_timem_object_tr DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC.
+
+  PUBLIC SECTION.
+
+    INTERFACES zif_timem_object.
+
+    METHODS constructor
+      IMPORTING
+        !id TYPE  trkorr.
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+
+    DATA id TYPE trkorr.
+
+    METHODS get_object_keys
+      RETURNING VALUE(result) TYPE trwbo_t_e071.
+
+    METHODS get_object
+      IMPORTING
+        object_key    TYPE trwbo_s_e071
+      RETURNING
+        VALUE(result) TYPE REF TO zif_timem_object.
+ENDCLASS.
+
+
+
+CLASS ZCL_TIMEM_OBJECT_TR IMPLEMENTATION.
+
+
+  METHOD constructor.
+    me->id = id.
+  ENDMETHOD.
+
+
+  METHOD get_object.
+    result = COND #(
+      WHEN object_key-pgmid = 'R3TR' AND ( object_key-object = 'CLAS' OR object_key-object = 'FUGR' )
+        OR object_key-pgmid = 'LIMU' AND ( object_key-object = 'FUNC' )
```

Comment:
```suggestion
        OR object_key-pgmid = 'LIMU' AND object_key-object = 'FUNC'
```

Reply:
Hi @larshp,
Thanks. `abaplint` is so great. Your suggestion was marked as outdated so I implemented it locally and committed it together with corrections to a couple of other `abaplint` warnings. Now `abaplint` is happy and I am too 😄